src/Backend/Modules/Settings/Actions/Index.php

```diff
@@ -352,20 +400,43 @@ private function validateForm(): void
                     'site_title_' . BL::getWorkingLanguage(),
                     $this->form->getField('site_title')->getValue()
                 );
+
+                if ($this->form->getField('google_tracking_google_analytics_tracking_id_enabled')->isChecked()) {
+                    $googleTrackingAnalyticsTrackingId = $this->form->getField('google_tracking_google_analytics_tracking_id')->getValue();
+                } else {
+                    $googleTrackingAnalyticsTrackingId = '';
+                }
+                $this->get('fork.settings')->set(
+                    'Core',
+                    'google_tracking_google_analytics_tracking_id',
+                    $googleTrackingAnalyticsTrackingId
+                );
+
+                if ($this->form->getField('google_tracking_google_tag_manager_container_id_enabled')->isChecked()) {
+                    $googleTrackingTagManagerContainerId = $this->form->getField('google_tracking_google_tag_manager_container_id')->getValue();
+                } else {
+                    $googleTrackingTagManagerContainerId = '';
+                }
                 $this->get('fork.settings')->set(
                     'Core',
-                    'site_html_header',
-                    $this->form->getField('site_html_header')->getValue()
+                    'google_tracking_google_tag_manager_container_id',
+                    $googleTrackingTagManagerContainerId
+                );
+
+                $this->get('fork.settings')->set(
+                    'Core',
+                    'site_html_head',
+                    $this->form->getField('site_html_head')->getValue()
                 );
                 $this->get('fork.settings')->set(
                     'Core',
-                    'site_start_of_body_scripts',
```

Comment:
I would write to the old one as well with a deprecation notice for BC reasons. There are people that upgrade to the latest version of fork and  this might create issues if they use that setting somewhere

Reply:
Fixed in dabf936ae918db1eac41e60b3cfe98587fc8d0aa